client/client.go

```diff
@@ -356,12 +354,11 @@ func (r *NotaryRepository) Publish(getPass passwordRetriever) error {
 
 	// check if our root file is nearing expiry. Resign if it is.
 	if nearExpiry(r.tufRepo.Root) || r.tufRepo.Root.Dirty {
-		passphrase, err := getPass()
```

Comment:
Looks like you've removed the one place we were using the PassphraseRetriever so unless I'm missing something, `Publish()` doesn't need to take it as an argument, which seems like it will make lots of people happy :-)